subsys/bluetooth/mesh/prov.c

```diff
@@ -1465,6 +1750,41 @@ void bt_mesh_pb_adv_recv(struct net_buf_simple *buf)
 
 	gen_prov_recv(&rx, buf);
 }
+
+#if defined(CONFIG_BT_MESH_PROVISIONER)
+int bt_mesh_pb_adv_open(const u8_t uuid[16], u16_t net_idx, u16_t addr,
+			u8_t attention_duration)
+{
+	BT_DBG("uuid %s", bt_hex(uuid, 16));
+
+	if (atomic_test_and_set_bit(link.flags, LINK_ACTIVE)) {
+		return -EBUSY;
+	}
+
+	link.node = bt_mesh_node_alloc(addr);
+	if (link.node == NULL) {
+		BT_ERR("Failed to allocate node");
+		reset_state();
+		return -ENOMEM;
+	}
+
+
+	atomic_set_bit(link.flags, PROVISIONER);
+
+	bt_rand(&link.id, sizeof(link.id));
+	link.tx.id = 0x7F;
+	link.node->net_idx = net_idx;
+	link.node->addr = addr;
```

Comment:
Doesn't this get initialised by `bt_mesh_node_alloc()`? I'd just pass both `addr` and `net_idx`to `bt_mesh_node_alloc()` and initialize both there.